content/microservices/mqtt.md

```diff
@@ -14,6 +14,8 @@ $ npm i --save mqtt
 
 To use the MQTT transporter, pass the following options object to the `createMicroservice()` method:
 
+> info **Hint** If you were to use MQTT in a docker environment I recommend using `hostname` as a parameter options.
```

Comment:
Why would we want to add this to the docs? This feels more like a comment that would belong in a blog post

Reply:
It might help partly as there is a little confusion between host and hostname, maybe I can rewrite the suggestion differently

Reply:
"In a docker environment it is recommended to use hostname instead of host"
Maybe this is how it could go?

Reply:
Possibly, but it still seems out of place, as there's not really anything else in the docs that mentions docker. This feels like it should be one of those things where understanding docker and what the properties means is more valuable than having documentation tell you what to set. Especially when with Nest it's just _using_ RabbitMQ and not setting it up directly.

Reply:
"In development environments where it is necessary to specify the port instead of host use hostname"